ResearchKit/Common/ORKScaleSliderView.m

```diff
@@ -128,79 +159,68 @@ - (void)updateConstraints {
     
     NSDictionary *views = NSDictionaryOfVariableBindings(_slider,_leftRangeLabel,_rightRangeLabel, _valueLabel);
     
-    if ([_formatProvider isVertical])
-    {
+    if ([_formatProvider isVertical]) {
```

Comment:
I kept the original updateConstraints there when I changed the constraints. You are correct that they are fixed constraints and updateConstraints is not needed.  I've eliminated it, and moved the code up into the initializer.